control/fixtures/fixtures.go

```diff
@@ -32,7 +32,7 @@ import (
 
 var (
 	PluginName = "snap-plugin-collector-mock2"
-	SnapPath   = os.Getenv("SNAP_PATH")
+	SnapPath   = os.ExpandEnv(os.Getenv("SNAP_PATH"))
```

Comment:
if you can put this as a "func" then calling it. It seems better than changing it all over the places. The advantage for simply change it in one place in the future.